ClearKeep/Scenes/Search/ViewModel/SearchMessageViewModel.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  SearchSearchMessageViewModel.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 16/08/2022.
+//
+
+import Foundation
+import CommonUI
+import Model
+import ChatSecure
+
+public struct SearchMessageViewModel: Identifiable, Equatable, IMessageViewModel {
+	public var id: String
+	public var groupId: Int64
+	public var groupType: String
+	public var fromClientId: String
+	public var fromClientName: String
+	public var clientId: String
+	public var message: String
+	public var dateCreated: Int64
+	public var isMine: Bool
+	public var isQuoteMessage: Bool
+	public var isForwardedMessage: Bool
+	public var isImageMessage: Bool
+	public var isFileMessage: Bool
+	public var clientWorkspaceDomain: String
+	public var avatar : String
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/ClearKeep/Scenes/Search/ViewModel/SearchMessageViewModel.swift#L28"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>SearchMessageViewModel.swift:28</code></td>
    </tr>
  </tbody>
</table>